heft-plugins/heft-webpack5-plugin/README.md

```diff
@@ -0,0 +1,13 @@
+# @rushstack/heft-webpack5-plugin
+
+> 🚨 _This is an early preview release. Please report issues!_ 🚨
+
+This is a Heft plugin for using Webpack 5 during the "bundle" stage.
+
+## Links
+
+- [CHANGELOG.md](https://github.com/microsoft/rushstack/blob/master/heft-plugins/heft-webpack5-plugin/CHANGELOG.md) - Find
+  out what's new in the latest version
+- [API Reference](https://rushstack.io/pages/api/heft-webpack5-plugin/)
```

Comment:
Plugins technically act as an "API" for other plugins to access.  The .api.md file is useful to generate, since it identifies issues such as your `ae-forgotten-export` warning.  But do you think it's worth including in the website API reference?  

Would that be actually meaningful to developers?  I feel like they would probably go straight to the `.d.ts` files.

Reply:
There are a few useful structures, but I can remove for now.